Forta-Bots-Challenge-2/README.md

```diff
@@ -0,0 +1,32 @@
+# Uniswap V3 Swap Detection Bot
+
+## Description
+
+This bot detects Swap events that are emitted by Uniswap V3 pools
+
+## Supported Chains
+
+- Ethereum
+
+## Alerts
+
+- NETHERMIND-1
+  - Fired when a Swap event is emitted by a Uniswap V3 pool
+  - Severity is always set to "low"
+  - Type is always set to "info"
+  - Metadata
+      - "sender": address of first participant
```

Comment:
```suggestion
      - `sender`: address of first participant
```
Whenever we add properties to the metadata, we use ticks (`). This would be applicable in the others as well.